src/nifgen/examples/nifgen_standard_function_grpc.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python
```

Comment:
I would expect `nifgen/examples.rst` but it's missing.

Reply:
Updated